src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -621,7 +606,7 @@ private static bool TryDeserializeJson<TResult>(string json, out TResult result)
         /// <summary>
         /// Performs required static initialization in the Edge context.
         /// </summary>
-        private static void Initialize()
+        private static async Task Initialize()
```

Comment:
Yes, there's more work, that's why at times it is beneficial to just return the task, but you need to understand the ramifications. In most cases, the added overhead is negligible and do not justify some of the issues that may arise from just returning the task. Principally in this code, since this will be executed only once and a cached task will be returned for every subsequent call.